src/ai/backend/manager/models/gql_models/container_registry.py

```diff
@@ -70,3 +73,94 @@ async def mutate(
             .where(AssociationContainerRegistriesGroupsRow.group_id == group_id)
         )
         return await simple_db_mutate(cls, info.context, delete_query)
+
+
+class CreateContainerRegistryQuota(graphene.Mutation):
+    """Added in 24.12.0."""
+
+    allowed_roles = (
+        UserRole.SUPERADMIN,
+        UserRole.ADMIN,
+    )
+
```

Comment:
@fregataa I have a question.
I think I heard that "ADMIN" refers to "Domain Admin." Is that correct?
If so, should only SuperAdmins be allowed to perform those mutations?
(Currently, there is no Project Admin role.)